src/test/java/com/teragrep/cfe_39/CombinedFullTest.java

```diff
@@ -0,0 +1,309 @@
+                    looper++;
+                } else if (looper == 12) {
+                    Assertions.assertEquals("{\"timestamp\": 1650872092242000, \"message\": \"25.04.2022 07:34:52.242 [ERROR] com.teragrep.jla_02.Log4j2 [instanceId=01, thread=Thread-0, userId=, sessionId=, requestId=, SUBJECT=, VERB=, OBJECT=, OUTCOME=, message=Log4j2 error daily says hi!]\", \"directory\": \"jla02logger\", \"stream\": \"test:jla02logger:0\", \"host\": \"jla-02.default\", \"input\": \"imrelp:cfe-06-0.cfe-06.default:\", \"partition\": \"" + partitionCounter + "\", \"offset\": 12, \"origin\": \"jla-02.default\"}", record.toString());
+                    looper++;
+                } else {
+                    Assertions.assertEquals("{\"timestamp\": 1650872092243000, \"message\": \"25.04.2022 07:34:52.243 [ERROR] com.teragrep.jla_02.Log4j2 [instanceId=01, thread=Thread-0, userId=, sessionId=, requestId=, SUBJECT=, VERB=, OBJECT=, OUTCOME=, message=Log4j2 error metric says hi!]\", \"directory\": \"jla02logger\", \"stream\": \"test:jla02logger:0\", \"host\": \"jla-02.default\", \"input\": \"imrelp:cfe-06-0.cfe-06.default:\", \"partition\": \"" + partitionCounter + "\", \"offset\": 13, \"origin\": \"jla-02.default\"}", record.toString());
+                    looper = 0;
+                    LOGGER.info("Partition {} passed assertions.", partitionCounter);
+                    partitionCounter++;
+                }
+            }
+            inputStream.close();
+        }
+        fs.close();
+    }
+
+    private void insertMockFiles() throws IOException {
+        String path = config.getHdfsPath()+"/"+"testConsumerTopic"; // "hdfs:///opt/teragrep/cfe_39/srv/testConsumerTopic"
+        String hdfsURI = config.getHdfsuri();
+        // ====== Init HDFS File System Object
+        Configuration conf = new Configuration();
+        // Set FileSystem URI
+        conf.set("fs.defaultFS", hdfsURI);
+        // Because of Maven
+        conf.set("fs.hdfs.impl", org.apache.hadoop.hdfs.DistributedFileSystem.class.getName());
+        conf.set("fs.file.impl", org.apache.hadoop.fs.LocalFileSystem.class.getName());
+        // Set HADOOP user
+        System.setProperty("HADOOP_USER_NAME", "hdfs");
+        System.setProperty("hadoop.home.dir", "/");
+        //Get the filesystem - HDFS
+        FileSystem fs = FileSystem.get(URI.create(hdfsURI), conf);
+
+
+        //==== Create directory if not exists
+        Path workingDir = fs.getWorkingDirectory();
+        // Sets the directory where the data should be stored, if the directory doesn't exist then it's created.
+        Path newDirectoryPath = new Path(path);
+        if (!fs.exists(newDirectoryPath)) {
```

Comment:
No conditionals in tests

Reply:
Solved in commit 65342c2e865912f981dfe82f470cab6884f72cdb and f67e59208bb4738dd503242db41f9c475b0e23a8